src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        int number = new Random().nextInt(101);
```

Comment:
replace 101 with a constant that equals 100
```suggestion
        int number = new Random().nextInt(101);
```